db/schema.rb

```diff
@@ -730,7 +730,7 @@
   create_table "temp_sites", force: :cascade do |t|
     t.string "code", null: false
     t.string "name", null: false
-    t.string "uuid"
+    t.string "uuid", null: false
```

Comment:
I don't think we can add this at this stage? This can come in after we perform our backfills successfully